cmd/client/main.go

```diff
@@ -202,6 +196,8 @@ func (c *Client) run(o *GrpcProxyClientOptions) error {
 	if err != nil {
 		return fmt.Errorf("failed to send request to client, got %v", err)
 	}
+	defer response.Body.Close() // TODO: proxy server should handle the case where Body isn't closed.
```

Comment:
If we aren't going to handle it in this PR, then we should file an issue.

Reply:
created #27